### PR TITLE
Issue 13725 add a dropdown to filter content type listing

### PR DESCRIPTION
**Pull request by hmoreras**

### CODE REVIEW

File: src/app/view/components/dot-base-type-selector/dot-base-type-selector.component.ts [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+import { map, take } from 'rxjs/operators';
+
+@Component({
+    selector: 'dot-base-type-selector',
+    templateUrl: './dot-base-type-selector.component.html',
+    styleUrls: ['./dot-base-type-selector.component.scss']
+})
+export class DotBaseTypeSelectorComponent implements OnInit {
+    @Input() value: SelectItem;
+    @Output() selected = new EventEmitter<string>();
```
Comment (freddyucv) [inline src/app/view/components/dot-base-type-selector/dot-base-type-selector.component.ts]: change is a better name

File: src/app/portlets/content-types/main/content-types.component.ts [hunk 2]
```diff
@@ -104,6 +104,11 @@ export class ContentTypesPortletComponent implements OnInit {
         });
     }
 
+    public changeBaseTypeSelector(value: string) {
+        this.listing.paginatorService.setExtraParams('type', value);
+        this.listing.loadDataPaginationEvent({first: 0});
```
Comment (freddyucv) [inline src/app/portlets/content-types/main/content-types.component.ts]: I think here is better this.listing.paginatorService.getFistPage()

Comment (hmoreras) [reply src/app/portlets/content-types/main/content-types.component.ts]: `this.listing.paginatorService.getFistPage()` do not load the new results in the list, when I call ` this.listing.loadDataPaginationEvent({first: 0});` it calls `loadData` witch update the items of the listing here: https://github.com/dotCMS/core-web/blob/9f4b1db28eb6082e62ec37b19edb957dc1b27326/src/app/view/components/listing-data-table/listing-data-table.component.ts#L90

File: src/app/view/components/dot-base-type-selector/dot-base-type-selector.component.ts (hunk 1 above)
Comment (hmoreras) [reply src/app/view/components/dot-base-type-selector/dot-base-type-selector.component.ts]: we have being using `selected` in other components like persona o device selector to emulate the "natural" behavior of a select in html.

**Pull request merged**